target/i386/cpu.h

```diff
@@ -2334,12 +2334,6 @@ static inline bool is_mmu_index_32(int mmu_index)
     return mmu_index & 1;
 }
 
-static inline bool is_mmu_index_32(int mmu_index)
```

Comment:
This is a different feature and should be in a different commit. 
In I fact I just reverted the commit to check if the test issue was related to the VLA change, but got that reverted at the same time... which broke the build...